packages/driver/src/cy/commands/cookies.ts

```diff
@@ -192,27 +180,27 @@ export default function (Commands, Cypress, cy, state, config) {
   }
 
   return Commands.addAll({
-    getCookie (name, userOptions: Partial<Cypress.Loggable & Cypress.Timeoutable> = {}) {
-      const options: InternalGetCookieOptions = _.defaults({}, userOptions, {
+    getCookie (name, userOptions: Partial<Cypress.CookieOptions> = {}) {
+      const options: Partial<Cypress.CookieOptions> = _.defaults({}, userOptions, {
         log: true,
       })
 
       const responseTimeout = options.timeout || config('responseTimeout')
 
       options.timeout = options.timeout || config('defaultCommandTimeout')
 
+      let cookie: Cypress.Cookie
```

Comment:
```suggestion
      const cookie: Cypress.Cookie = option.cookie
```

Reply:
NVM...thats the console props 🤦🏻‍♀️ its in the automation request below.

Reply:
This is set below by the result of `automateCookies('get:cookie')`. There is no `options.cookie`.